spec/models/bill_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Bill, type: :model do
+  describe 'associations' do
+    it { should belong_to(:author).class_name('User') }
+  end
```

Comment:
- Hey, let's also add tests that show the bill has a name and an amount as in the bill model.


![image](https://github.com/lukman155/Budget-Rails-Capstone/assets/59397159/cd8617ae-9e2a-4512-b39d-93fc82cbe77b)